### ISSUE TITLE
Quick Start documentation: missing images

### ISSUE THREAD

**Issue by quantumm** [PyMoDAQ/PyMoDAQ#421]
In the [Quick Start](https://pymodaq.cnrs.fr/en/5.0.x_dev/quick_start.html), Fig. 1.6 to 1.8 links are broken.

**Comment by seb5g** [comment]
could you repair that?

**Comment by quantumm** [comment]
Yes it is in my TODO list ;)

**Comment by seb5g** [comment]
It should be in the top of your todo list now :-)

**Comment by malik-irain** [comment]
While you are adding image from lab stories you should add these one too!

**Comment by quantumm** [comment]
Yes after the lab story PR I plan to clean the documentation a bit. Let it open for now ;)

**Comment by quantumm** [comment]
This has been solved in #504 .
